README.md

```diff
@@ -37,3 +43,31 @@ You can access the documentation on your machine by running the following in any
 cargo doc --package tikv-client --open
 # If it didn't work, browse file URL it tried to open with your browser.
 ```
+
+## Toolchain versions
+
+To check what version of Rust you are using, run
+
+```bash
+rustc --version
+```
+
+you'll see something like `rustc 1.36.0-nightly (a784a8022 2019-05-09)` where the `1.36.0` is the toolchain version, and `nightly` is the channel (stable/beta/nightly). To install another toolchain use

```

Comment:
These sentences have inconsistent capitalization. ('To build', '(where `nightly`', 'you'll see')